core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -181,23 +181,23 @@ private[history] class FsHistoryProvider(conf: SparkConf, clock: Clock)
     processing.remove(path.getName)
   }
 
-  private val blacklist = new ConcurrentHashMap[String, Long]
+  private val ignoreList = new ConcurrentHashMap[String, Long]
 
   // Visible for testing
-  private[history] def isBlacklisted(path: Path): Boolean = {
-    blacklist.containsKey(path.getName)
+  private[history] def isIgnored(path: Path): Boolean = {
+    ignoreList.containsKey(path.getName)
   }
 
-  private def blacklist(path: Path): Unit = {
-    blacklist.put(path.getName, clock.getTimeMillis())
+  private def ignore(path: Path): Unit = {
+    ignoreList.put(path.getName, clock.getTimeMillis())
   }
 
   /**
-   * Removes expired entries in the blacklist, according to the provided `expireTimeInSeconds`.
+   * Removes expired entries in the ignoreList, according to the provided `expireTimeInSeconds`.
    */
-  private def clearBlacklist(expireTimeInSeconds: Long): Unit = {
+  private def clearIgnoreList(expireTimeInSeconds: Long): Unit = {
     val expiredThreshold = clock.getTimeMillis() - expireTimeInSeconds * 1000
-    blacklist.asScala.retain((_, creationTime) => creationTime >= expiredThreshold)
+    ignoreList.asScala.retain((_, creationTime) => creationTime >= expiredThreshold)
```

Comment:
since this is internal not as big of a deal but not sure if we want to come to consensus on https://issues.apache.org/jira/browse/SPARK-32037 terminology first.  Here ignorelist seems ok, although blocklist or notallowed (since its an access control exception) might fit better.

Reply:
I think the discussion on SPARK-32037 won't necessarily apply here -- the discussion there, IMO, isn't what the one best term to replace "blacklist" is, but rather what the best term to describe the "blacklisting feature" is. The best term to replace "blacklist" in each of these scenarios will vary, independent of that discussion.

For this specific one, I agree that `ignoreList` is not great. I struggled with this one. What do you think of `inaccessibleList`? I think this is the most descriptive. `blockList` IMO doesn't really describe what is happening.

Reply:
"unhealthy" to me doesn't fit this situation.  We are just keeping a list of event logs that can't be read.  I'm fine with inaccessibleList

Reply:
Agreed, nothing is "wrong"/"unhealthy" with the file, it is just not accessible to us. I'll go with `inaccessible`